dist/drawer/ds6/drawer.css

```diff
@@ -91,10 +91,6 @@
 .drawer__window {
   display: -webkit-box;
   display: flex;
-  -webkit-box-orient: vertical;
```

Comment:
Is this change intentional? Looks like it has broken the drawer:

<img width="577" alt="Screen Shot 2020-05-05 at 9 05 33 PM" src="https://user-images.githubusercontent.com/38065/81137711-929b7180-8f14-11ea-9288-dfd41e0ca98d.png">

Reply:
I searched for `.dialog-window()` and it did not give a usage in drawer (As it uses `.dialog-window(100%)`) so missed updating `drawer` module. Thanks for catching this bug. This should be fixed now.